concourse/scripts/check-certificates.rb

```diff
@@ -68,11 +68,13 @@
 unless expiring_certificate_names.empty?
   separator
 
-  puts "The following certificates are expiring and require operator intervention:"
+  puts "The following certificates are expiring very soon:"
 
   expiring_certificate_names.each do |cert|
     puts cert.red
   end
+
+  puts "\nThe deployment pipeline must be re-run from the beginning before these certificates have a chance to expire."
```

Comment:
Is it possible to explain why? The more information we can provide here the better, honestly. It's a scary thing to have go awry.

Reply:
The problem is, there are multiple possible cases in which this error could be shown and I don't want to say anything that would be incorrect in any cases.

Reply:
In fact I've noticed that it could happen that this message is triggered on the _first_ stage of the process, in which case re-running the pipeline would show you the same message again. So I'm going to have to provision for that in the message too.